packages/salesforcedx-utils-vscode/src/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const SETTING_CLEAR_OUTPUT_TAB = 'clearOutputTab';
```

Comment:
These are now all exported by salesforcedx-utils-vscode and consumed in other modules so that the string 'salesforcedx-vscode-core' is not in 20 places.